x/fungible/types/message_update_zrc20_withdraw_fee.go

```diff
@@ -0,0 +1,53 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+const TypeMsgUpdateZRC20WithdrawFee = "update_zrc20_withdraw_fee"
+
+var _ sdk.Msg = &MsgUpdateZRC20WithdrawFee{}
+
+func NewMsgUpdateZRC20WithdrawFee(creator string, zrc20 string, newFee sdk.Uint) *MsgUpdateZRC20WithdrawFee {
```

Comment:
`sdk.Uint` has been deprecated in favor of using the new library for math: `cosmossdk.io/math`

Reply:
yeah, but sdk.Uint is used in a lot of places. Let's put up a separate issue/PR to address this.